pkg/controller/cloneset/sync/cloneset_sync_utils.go

```diff
@@ -116,7 +112,7 @@ func calculateDiffsWithExpectation(cs *appsv1alpha1.CloneSet, pods []*v1.Pod, cu
 
 				if isSpecifiedDelete(cs, p) {
 					toDeleteNewRevisionCount++
-				} else if !isPodReady(coreControl, p, cs.Spec.MinReadySeconds) {
```

Comment:
why drop the minreadyseconds check?

Reply:
Because it calculates the count of not-ready pods, not unavailable pods.